urdf/include/urdf/model.h

```diff
@@ -56,6 +57,7 @@ class Model: public ModelInterface
   bool initFile(const std::string& filename);
   /// \brief Load Model given the name of a parameter on the parameter server
   bool initParam(const std::string& param);
+  bool initParamWithNodeHandle(const std::string& param, const ros::NodeHandle& nh);
```

Comment:
I'm not sure that this is doing the same thing as the kinetic-devel version.  There, nh is always initialized with a new ros::NodeHandle() object based on the function signature.  Since we aren't doing that here (why not?), in the case where someone just calls initparamWithNodeHandle(), nh is undefined.  Am I missing something?